### DISCUSSION TITLE
[Feature Request] Support for HTTP3

### DISCUSSION THREAD

**Discussion by x2q** [nginx-proxy/nginx-proxy#1873] [Ideas]
This [blog post](https://www.cloudflare.com/learning/performance/what-is-http3/) from Cloudflare outlines the benefits of using HTTP3 and the [current adoption of HTTP3 is shown here](https://w3techs.com/technologies/details/ce-http3#:~:text=HTTP%2F3%20is%20the%20third,derived%20from%20the%20QUIC%20protocol.&text=HTTP%2F3%20is%20now%20supported,from%202%25%20six%20months%20ago.).

**Comment by buchdag** [comment]
Hi. What's the current status of QUIC and HTTP/3 support in NGINX ?

**Reply by x2q** [reply]
The NGINX team has this post on HTTP3 support: https://www.nginx.com/blog/our-roadmap-quic-http-3-support-nginx/

**Reply by buchdag** [reply]
> The initial code commits are underway and we'll drop an update when the first mainline release has QUIC. We'll also be sure to Tweet from [@nginx](https://twitter.com/nginx) and [@nginxorg](https://twitter.com/nginxorg) to announce QUIC+HTTP/3 milestones.

http://disq.us/p/2leljk8

No announcement so far.

**Comment by buchdag** [comment]
@x2q discussion about QUIC and HTTP/3 support : #2271
